### DISCUSSION TITLE
Theme is breaking bin/magento commands

### DISCUSSION THREAD

**Discussion by vagyvin** [breezefront/community#119] [1. Help]
composer require swissup/breeze-evolution
Warning from https://repo.packagist.org: Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^2.1 for swissup/breeze-evolution
./composer.json has been updated
Loading composer repositories with package information
Warning from https://repo.packagist.org: Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies (including require-dev)
Package operations: 6 installs, 0 updates, 0 removals
  - Installing swissup/module-rtl (1.3.6): Downloading (100%)
  - Installing swissup/module-breeze (2.5.5): Downloading (100%)
  - Installing swissup/module-breeze-integrations (1.3.1): Downloading (100%)
  - Installing swissup/theme-frontend-breeze-blank (2.3.6): Downloading (100%)
  - Installing swissup/theme-frontend-breeze-evolution (2.1.2): Downloading (100%)
  - Installing swissup/breeze-evolution (2.1.2)
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
Package laminas/laminas-console is abandoned, you should avoid using it. Use laminas/laminas-cli instead.
Package mageplaza/module-sitemap is abandoned, you should avoid using it. No replacement was suggested.
Package mirasvit/module-blog is abandoned, you should avoid using it. No replacement was suggested.
Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
Package phpunit/phpunit-mock-objects is abandoned, you should avoid using it. No replacement was suggested.
Package sebastian/finder-facade is abandoned, you should avoid using it. No replacement was suggested.
Package sebastian/phpcpd is abandoned, you should avoid using it. No replacement was suggested.
Package theseer/fdomdocument is abandoned, you should avoid using it. No replacement was suggested.
Package true/punycode is abandoned, you should avoid using it. No replacement was suggested.
Package yubico/u2flib-server is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating autoload files

sudo bin/magento setup:upgrade --safe-mode=1
Cache cleared successfully
File system cleanup:
/var/www/html/generated/code/Amasty
/var/www/html/generated/code/Composer
/var/www/html/generated/code/MSP
/var/www/html/generated/code/Magento
/var/www/html/generated/code/Mageplaza
/var/www/html/generated/code/Mirasvit
/var/www/html/generated/code/Swissup
/var/www/html/generated/code/Symfony
/var/www/html/generated/code/WebPanda
The directory '/var/www/html/generated/metadata/' doesn't exist - skipping cleanup
Updating modules:

**Comment by roma84** [comment]
Do not see the error message here, looks like your log is cut.

**Comment by vagyvin** [comment]
[2023-05-03 06:34:03] main.CRITICAL: Type Error occurred when creating object: Swissup\Marketplace\Model\Channel\Composer, Argument 9 passed to Swissup\Marketplace\Model\Channel\Composer::__construct() must be an instance of Magento\Framework\HTTP\ClientFactory, instance of Magento\Framework\HTTP\ZendClientFactory given, called in /var/www/html/vendor/magento/framework/ObjectManager/Factory/AbstractFactory.php on line 121

**Reply by vovayatsyuk** [reply]
You get this error because of outdated info about `Swissup\Marketplace\Model\Channel\Composer` somewhere in the magento cache. Try to cleanup magento folders manually:

```
rm -rf var/cache generated/code generated/metadata
```

p.s. If you are using redis of something similar, you need to clean it using corresponding command.

**Comment by vagyvin** [comment]
After that none of the bin/magento commands work.
Tried bin/magento setup:upgrade -vvv ->>> It does nothing
